### ISSUE TITLE
babel-preset-ts-node6-bluebird issue

### ISSUE THREAD

**Issue by tylerlong** [develar/lazy-val#2]
It seems that project depending on this project (either directly or indirectly) have to install babel-preset-ts-node6-bluebird, otherwise webpack will complain.

Can we remove the following from package.json:

```
"babel": {
    "presets": [
      "babel-preset-ts-node6-bluebird"
    ]
  },
```

?

**Comment by develar** [comment]
No, or do you use src directly instead of published on npm (generated out)?

**Comment by tylerlong** [comment]
I use the npm library instead of source.

It seems that webpack or babel interprets that piece of code and it required every project which uses this library to install babel-preset-ts-node6-bluebird.   Which doesn't make much sense because not every project uses TS and bluebird.

**Comment by tylerlong** [comment]
For babel presets, I've seen people put them in to `.babelrc` or `webpack.config.js`. This is the only project(I've seen) that put them into `package.json`

**Comment by tylerlong** [comment]
And by the way, you are writing TS, why do you need babel? TS itself is a transcompiler.
